script.js

```diff
@@ -2,6 +2,37 @@ const cards = document.querySelectorAll(".card");
 let matchedPairs = 0;
 let cardOne, cardTwo;
 let disableDeck = false;
+let timer = document.querySelector('.timer');
+let second, minute, interval;
+let counter = document.querySelector('.moves');
+let displayCongratulations = document.getElementById('congratulations');
+let modalContent = document.querySelector('.modal-content');
+
+function movesCounter() {
```

Comment:
It doesn't look like this movesCounter function is used. It should be called once to start the clock and counter when the page loads. Then it should be called every time `flipCard` is called.